perun-oidc-server/src/main/java/cz/muni/ics/oidc/server/claims/ClaimUtils.java

```diff
@@ -27,6 +27,34 @@ public static String fillStringPropertyOrNoVal(String suffix, ClaimModifierInitC
         return fillStringPropertyOrNoVal(ctx.getProperty(suffix, NO_VALUE));
     }
 
+    public static String fillStringMandatoryProperty(String suffix, ClaimSourceInitContext ctx, String claimName) {
+        String filled = fillStringPropertyOrNoVal(ctx.getProperty(suffix, NO_VALUE));
+
+        if (filled == null) {
+            throw new IllegalArgumentException(claimName + " - missing mandatory configuration option: " + suffix);
+        }
+
+        return filled;
+    }
+
+    public static String fillStringMandatoryProperty(String suffix, ClaimModifierInitContext ctx, String claimName) {
+        String filled = fillStringPropertyOrNoVal(ctx.getProperty(suffix, NO_VALUE));
+
+        if (filled == null) {
+            throw new IllegalArgumentException(claimName + " - missing mandatory configuration option: " + suffix);
+        }
+
+        return filled;
+    }
+
+    public static String fillStringPropertyOrDefaultVal(String suffix, ClaimSourceInitContext ctx, String defaultVal) {
```

Comment:
It would be a good idea to create a parent class for the `ClaimSourceInitContext` and `ClaimSourceModifierContext`, which could hold the common properties and methods. As a result, these methods would not need to be duplicated due to different types of `ctx` parameters.